src/css/intlTelInput.scss

```diff
@@ -229,6 +234,41 @@ $flagsImageExtension: "png" !default;
     height: 100%;
     opacity: 0;
   }
+  
+  // right-to-left
+  &.iti-rtl {
+    direction: rtl;
+    
+    input, input[type=text], input[type=tel] {
+      // cancel left padding and set it as right padding
+      padding-left: 0;
+      padding-right: $selectedFlagWidth + $inputPadding;
+      
+      // for RTL languages, the phone number should be LTR and right-aligned
+      direction: ltr !important;
+      text-align: right;
+    }
+    
+    .selected-flag {
+      padding-left: 0;
+      
+      .iti-flag {
+        left: $flagPadding;
+      }
+    }
+    
+    .country-list {
+      .flag, .country-name {
+        // cancel right margin and set it as left margin
+        margin-left: $countryListSpacing;
+        margin-right: 0;
+      }
+      .dial-code {
+        // dial codes should be LTR
+        direction: ltr;
+      }
+    }
+  }
 }
```

Comment:
Not sure how this could be automated to be honest.
It's very custom, so to speak.
If you believe the project you linked could do this, then great, close the PR unmerged and do it.